common/logisticspipes/blocks/powertile/LogisticsPowerJunctionTileEntity.java

```diff
@@ -404,9 +432,12 @@ public boolean hasCapability(Capability<?> capability, @Nullable EnumFacing faci
 	@Nullable
 	@Override
 	public <T> T getCapability(Capability<T> capability, @Nullable EnumFacing facing) {
-		if(capability == CapabilityEnergy.ENERGY) {
+		if (capability == CapabilityEnergy.ENERGY) {
```

Comment:
same as above

Reply:
I actually need the null check here otherwise it will try to access the field if you pass null as capability argument. Not sure if that isn't undefined behavior though, tbh

Reply:
Why is it bad to check if null equals to any of these fields?

Reply:
Because if the capability parameter was null, and BC was not installed, the MJ capability fields also would be null, and the condition would be true which would make it try to access the mjReceiver field which also wouldn't exist. Now the field is just Object since I just cast it anyway, so no problem anymore